validators/performance/aiperf-bench.Dockerfile

```
@@ -30,7 +30,11 @@ ARG AIPERF_VERSION=0.7.0
 # endpoint, no filesystem writes outside /tmp, and no privileged ops —
 # running as a dedicated non-root user hardens the image for air-gap /
 # multi-tenant deployments.
-RUN pip install --no-cache-dir "aiperf==${AIPERF_VERSION}" \
+#
+# Upgrade pip first so the install uses a pip with current CVE fixes
+# (the base image ships an older pip pinned to its release date).
+RUN pip install --no-cache-dir --upgrade pip \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)